"GET /static/css/site.css HTTP/1.1" 404 1660
Internal Server Error: /
Traceback (most recent call last):
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 303, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such column: dsc_dsc.device_name_list

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 35, in inner
    response = get_response(request)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Study/opensourcedev/Harness-Controller/DSCHA_Harness/dashboard/views.py", line 7, in index
    {})
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 175, in render
    return self._render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/loader_tags.py", line 155, in render
    return compiled_parent._render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/loader_tags.py", line 194, in render
    return template.render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 177, in render
    return self._render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 167, in _render
    return self.nodelist.render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 943, in render
    bit = node.render_annotated(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/base.py", line 910, in render_annotated
    return self.render(context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/template/defaulttags.py", line 168, in render
    len_values = len(values)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/models/query.py", line 254, in __len__
    self._fetch_all()
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/models/query.py", line 1179, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 1066, in execute_sql
    cursor.execute(sql, params)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 100, in execute
    return super().execute(sql, params)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 68, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 77, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/utils.py", line 89, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 85, in _execute
    return self.cursor.execute(sql, params)
  File "/Study/opensourcedev/Harness-Controller/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 303, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such column: dsc_dsc.device_name_list
"GET / HTTP/1.1" 500 203062
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 4346
